deep_quoridor/src/agents/greedy.py

```diff
@@ -117,7 +127,7 @@ def _shortest_path_from_me(self, game: Quoridor, action_mask: np.ndarray, target
         for move in first_moves:
             path = self._shortest_path_from(game, move, target_row)
             if shortest_path is None or len(path) < len(shortest_path):
-                shortest_path = [move] + path
+                shortest_path = [my_position] + path
```

Comment:
now tha path includes the initial position, which is "move", so instead we need to include the current pawn position